hexrd/ui/import_data_panel.py

```diff
@@ -427,28 +429,29 @@ def completed(self):
         self.check_for_unsaved_changes()
 
         files = []
-        detectors = self.detector_defaults['default_config'].get(
+        detectors = self.detector_defaults['default_config'].setdefault(
             'detectors', {})
         not_set = [d for d in detectors if d not in self.completed_detectors]
         for det in not_set:
-            del(detectors[det])
+            del(self.detector_defaults['default_config']['detectors'][det])
 
+        instr = HEDMInstrument(
+            instrument_config=self.detector_defaults['default_config'])
         for det in self.completed_detectors:
-            transform = detectors[det].setdefault('transform', {})
-            *zx, z = transform['tilt']
-            transform['tilt'] = (
-                [*zx, (z + float(self.edited_images[det]['tilt']))])
-            panel_buffer = detectors[det].setdefault('panel_buffer', [])
-            panel_buffer = self.edited_images[det]['panel_buffer']
+            panel = instr.detectors[det]
+            *zx, z = panel.tilt
```

Comment:
you'll need to import the following methods:
```
from hexrd.rotations import angleAxisOfRotMat, make_rmat_euler
```

then replace 442-443 with the following:
```
# first need the zxz Euler angles from the panel rotation matrix.
*zx, z = angles_from_rmat_zxz(panel.rmat)

# convert updated zxz angles to rmat
rmat_updated = make_rmat_euler(
    [*zx, (z + float(self.edited_images[det]['tilt']))],
    'zxz', extrinsic=False
)

# convert to angle-axis parameters
rang, raxs = angleAxisOfRotMat(rmat_updated)

# update tilt property on panel
panel.tilt = rang*raxs.flatten()
```

Reply:
This has been added in, thanks for the help!